package.json

```diff
@@ -30,4 +30,4 @@
         "typescript": "^2.5.2",
         "vsce": "^1.41.0"
     }
-}
\ No newline at end of file
+}
```

Comment:
OK… xD You keep removing newlines in some files and adding them back in others. Your editor might be doing weird things.